src/integrationTest/groovy/wooga/gradle/build/UnityBuildPluginIntegrationSpec.groovy

```diff
@@ -148,9 +148,9 @@ class UnityBuildPluginIntegrationSpec extends UnityIntegrationSpec {
         } else if (location == "properties") {
             createFile("gradle.properties") << "${propertiesKey}=${value}"
         } else if (location == "extension") {
-            buildFile << "${extensionKey}('${value}')"
+            buildFile << "${extensionKey}('${escapedPath(value)}')"
```

Comment:
Where does `escapedPath` come from? Is it from parent class?

Reply:
Yes in `IntegrationSpec`